<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #404244;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>398</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <layout class="QHBoxLayout" name="logo">
       <item>
        <widget class="QLabel" name="label_logo">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/img/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;RUN TIME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="statusTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
</string>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="styleSheet">
       <string notr="true">QLabel {
  color: white; 
}

</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="situation_arg1">
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: white; 
}


</string>
         </property>
         <property name="text">
          <string>Enter catet a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="catet_a">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>22</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: white; 
}
QSpinBox {
color: white;
}

</string>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="situation_arg2">
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: white; 
}

</string>
         </property>
         <property name="text">
          <string>Enter catet b</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="catet_b">
         <property name="styleSheet">
          <string notr="true">QSpinBox {
color: white;
}
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: white; 
}

</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="INFO">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width:  100px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;

}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
          </property>
          <property name="text">
           <string>Result =</string>
          </property>
         </widget>
         <widget class="QLabel" name="printf_result">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width:  100px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;

}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="consider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width:  100px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;
color: black;
}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
         </property>
         <property name="text">
          <string>Сalculate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <item>
          <property name="text">
           <string>Bubble sorting</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>****** sorting</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Merge sorting</string>
          </property>
         </item>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>165</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width: 300px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;
color: black;
}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
         </property>
         <property name="text">
          <string>+1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width:  100px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;
color: black;
}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
         </property>
         <property name="text">
          <string>new window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settings">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width:  100px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;
color: black;
}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="test_na_tebya">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width:  100px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;
color: black;
}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
         </property>
         <property name="text">
          <string>Тест на пидора</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: #3B6582;
  color: white; 
 width:  100px;
 height: 25px;
  border: 1px solid gray;
}


QPushButton:hover {
    background-color: #E1E8EE;
color: black;
}
QPushButton:pressed {  background-color rgb(220, 116, 45) 
}</string>
         </property>
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar:hover {
color:black;
}
QMenuBar {
color: white;
 }
</string>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionRussian"/>
     <addaction name="actionEnglish"/>
     <addaction name="action_2"/>
    </widget>
    <widget class="QMenu" name="menuFunction">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Function</string>
     </property>
     <addaction name="actionPythagorean_theorem"/>
     <addaction name="actionFactorial"/>
     <addaction name="actionBubble_sorting"/>
     <addaction name="actionMerge_sorting"/>
     <addaction name="actionSorting_for_daun"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
    <addaction name="menuFunction"/>
    <addaction name="actionCloud_sichronize"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
  </widget>
  <action name="actionCloud">
   <property name="icon">
    <iconset>
     <normaloff>img/291255.png</normaloff>img/291255.png</iconset>
   </property>
   <property name="text">
    <string>Cloud</string>
   </property>
  </action>
  <action name="actionCloud_sichronize">
   <property name="icon">
    <iconset resource="yahzwhatisit.qrc">
     <normaloff>:/new/prefix1/img/291255.png</normaloff>:/new/prefix1/img/291255.png</iconset>
   </property>
   <property name="text">
    <string>Cloud sichronize</string>
   </property>
  </action>
  <action name="actionPythagorean_theorem">
   <property name="icon">
    <iconset resource="yahzwhatisit.qrc">
     <normaloff>:/new/prefix1/img/pifagor.jpg</normaloff>:/new/prefix1/img/pifagor.jpg</iconset>
   </property>
   <property name="text">
    <string>Pythagorean theorem</string>
   </property>
  </action>
  <action name="actionFactorial">
   <property name="icon">
    <iconset resource="yahzwhatisit.qrc">
     <normaloff>:/new/prefix1/img/fact.png</normaloff>:/new/prefix1/img/fact.png</iconset>
   </property>
   <property name="text">
    <string>Factorial</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="icon">
    <iconset resource="yahzwhatisit.qrc">
     <normaloff>:/new/prefix1/img/rus.png</normaloff>:/new/prefix1/img/rus.png</iconset>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="yahzwhatisit.qrc">
     <normaloff>:/new/prefix1/img/usa.jpg</normaloff>:/new/prefix1/img/usa.jpg</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Пидорский</string>
   </property>
  </action>
  <action name="actionBubble_sorting">
   <property name="icon">
    <iconset resource="yahzwhatisit.qrc">
     <normaloff>:/new/prefix1/img/bubble.png</normaloff>:/new/prefix1/img/bubble.png</iconset>
   </property>
   <property name="text">
    <string>Bubble sorting</string>
   </property>
  </action>
  <action name="actionMerge_sorting">
   <property name="text">
    <string>Merge sorting</string>
   </property>
  </action>
  <action name="actionSorting_for_daun">
   <property name="text">
    <string>Sorting for daun</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="yahzwhatisit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
